examples/games/loading_screen.rs

```diff
@@ -2,17 +2,9 @@
 use bevy::{ecs::system::SystemId, prelude::*};
 use pipelines_ready::*;
 
-// The way we'll go about doing this in this example is to
-// keep track of all assets that we want to have loaded before
-// we transition to the desired scene.
-//
-// In order to ensure that visual assets are fully rendered
```

Comment:
I don't think this comment should be removed in this PR.

Reply:
I don't exactly remember deleting this (in fact I vividly remember moving the constants above it), but I reinstated the comment in my most recent commit